feature/quicksettings/src/main/res/values/strings.xml

```diff
@@ -17,33 +17,27 @@
 <resources>
     <string name="quick_settings_front_camera_text">FRONT</string>
     <string name="quick_settings_back_camera_text">BACK</string>
+    <string name="quick_settings_front_camera_description">Front Camera</string>
+    <string name="quick_settings_back_camera_description">Back Camera</string>
 
     <string name="quick_settings_aspect_ratio_3_4">3:4</string>
     <string name="quick_settings_aspect_ratio_9_16">9:16</string>
     <string name="quick_settings_aspect_ratio_1_1">1:1</string>
-
-    <string name="quick_settings_flash_off">OFF</string>
-    <string name="quick_settings_flash_auto">AUTO</string>
-    <string name="quick_settings_flash_on">ON</string>
-
-    <string name="quick_settings_timer_off">OFF</string>
-    <string name="quick_settings_timer_3"></string>
-    <string name="quick_settings_timer_10"></string>
-
-    <string name="quick_settings_dropdown_description">Quick settings drop down</string>
-
-    <string name="quick_settings_front_camera_description">Front Camera</string>
-    <string name="quick_settings_back_camera_description">Back Camera</string>
-
     <string name="quick_settings_aspect_ratio_3_4_description">3 to 4 aspect ratio</string>
     <string name="quick_settings_aspect_ratio_9_16_description">9 to 16 aspect ratio</string>
     <string name="quick_settings_aspect_ratio_1_1_description">1 to 1 aspect ratio</string>
 
+    <string name="quick_settings_flash_off">OFF</string>
+    <string name="quick_settings_flash_auto">AUTO</string>
+    <string name="quick_settings_flash_on">ON</string>
     <string name="quick_settings_flash_off_description">Flash off</string>
     <string name="quick_settings_flash_auto_description">Auto flash</string>
     <string name="quick_settings_flash_on_description">Flash on</string>
 
-    <string name="quick_settings_timer_off_description">Timer off</string>
-    <string name="quick_settings_timer_3_description">3 seconds timer</string>
-    <string name="quick_settings_timer_10_description">10 seconds timer</string>
+    <string name="quick_settings_capture_mode_single">Single Stream Capture</string>
```

Comment:
nit: Let's drop "Capture". It doesn't add much context and the quick setting looks a little verbose.